plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseCaseInsensitiveMappingTest.java

```diff
@@ -348,4 +353,16 @@ private AutoCloseable withTable(String remoteTableName, String definition)
     {
         return withTable(getSession().getSchema().orElseThrow(), remoteTableName, definition);
     }
+
+    private void updateRuleBasedIdentifierMappingFileAndWait(Path file, List<SchemaMappingRule> schemaMappingRules, List<TableMappingRule> tableMappingRules)
+    {
+        try {
+            updateRuleBasedIdentifierMappingFile(file, schemaMappingRules, tableMappingRules);
```

Comment:
All users of `io.trino.plugin.jdbc.mapping.RuleBasedIdentifierMappingUtils#updateRuleBasedIdentifierMappingFile` are affected, so the sleep belongs there.

(note that all users = this class)

Reply:
Ah, I forgot to notice that the Utils is test-only.